management-microservice/src/main/java/com/nearsoft/fgaribay/mgmt/microservice/ManagementMicroserviceApplication.java

```diff
@@ -0,0 +1,21 @@
+package com.nearsoft.fgaribay.mgmt.microservice;
+
+import com.nearsoft.fgaribay.mgmt.config.BrokerConfig;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@SpringBootApplication
+@EntityScan(basePackages = {"com.nearsoft.fgaribay.mgmt.model"})
```

Comment:
Use `basePackageClasses` instead of `basePackage` to avoid harcoding package names.